meshtastic/interfaces/ble/lifecycle_service.py

```
@@ -0,0 +1,797 @@
+                source,
+            )
+            with iface._state_lock:
+                return (
+                    iface.auto_reconnect
+                    and not iface._closed
+                    and not iface._state_manager._is_closing
+                )
+
+        disconnect_lock_released = False
+        target_client = client
+        previous_client: "BLEClient | None" = None
```

Ruff (0.15.5)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)